### DISCUSSION TITLE
Is there a way to jump between two files like `ctrl-^`?

### DISCUSSION THREAD

**Discussion by cem-al** [VSCodeVim/Vim#8563] [Q&A]
I often use `ctrl-^` to switch between the current and last file, my vim binding looks like this:

`﻿map <SPACE><SPACE> <c-6>`

Is there a way to do this with VSCodeVim?

**Comment by OverRevvv** [comment]
### Try `ctrl + tab` its the default one but it does let the user go back and forth between 2 target files, 

### in other words `last file` and `current file`.
